python/tk_unreal/unreal_sg_engine.py

```diff
@@ -147,7 +148,7 @@ def _execute_deferred(self, callback):
         """
         unreal.log("{0} _execute_deferred called with {1}".format(self, callback.__str__()))
         self._callback = callback
-        
+
         from sgtk.platform.qt5 import QtCore
```

Comment:
Is ok to do explicit import?

Reply:
I don't know if this was done on purpose or not, and I don't know the potential side effects of changing it.